.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   jobs:
     - MOLECULE_DISTRO=debian:stretch-slim
```

Comment:
When adding bullseye to test matrix we're removing stretch from it.

Reply:
We need to maintain for a while support for Debian Stretch :-|

Reply:
We reached the conclusion to terminate the support given to Stretch in this role. Debian buster support should be enough. Stretch and buster are very similar so we can assume that if the role works for Buster, it should work for Stretch.